Tests/Marketplace/marketplace_services.py

```diff
@@ -63,6 +63,7 @@ class GCPConfig(object):
                        "ExportIndicators",
                        "Malware",
                        "DefaultPlaybook",
+                       "Access Investigation",
```

Comment:
It must be pack folder name `AccessInvestigation`

Reply:
```suggestion
                       "AccessInvestigation",
```